src/Directory.Build.props

```diff
@@ -8,6 +8,9 @@
 
     <DefineConstants Condition="'$(BUILDING_VSIX)' == 'true'">$(DefineConstants),BUILDING_VSIX</DefineConstants>
     <DefineConstants Condition="'$(LEGACY_CODE_METRICS_MODE)' == 'true'">$(DefineConstants),LEGACY_CODE_METRICS_MODE</DefineConstants>
+
```

Comment:
You probably either want to remove this empty line or move the following entries to a different `PropertyGroup`.

Reply:
Sure, let me do the latter and add a comment for clarity.